packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -271,31 +267,31 @@ class FlatTableState<T> extends State<FlatTable<T>>
   void _sortDataAndUpdate(
     ColumnData column,
     SortDirection direction, {
-    ColumnData secondarySortColumn,
+    ColumnData? secondarySortColumn,
   }) {
     setState(() {
       sortData(column, direction, secondarySortColumn: secondarySortColumn);
       if (widget.onSortChanged != null) {
-        widget.onSortChanged(
-          column,
+        widget.onSortChanged!(
+          column as ColumnData<T>,
```

Comment:
If you add the generic type to the `column` and `secondarySortColumn` parameters, can you get rid of the inline type casts for both?

Reply:
done